src/services/EdFi.DataManagementService.Api.E2E.Tests/EdFi.DataManagementService.Api.E2E.Tests.csproj

```diff
@@ -0,0 +1,44 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
I find Microsoft's choice of property name very confusing. This setting does the opposite of what it looks like: it makes it so that reference types are _not nullable_. Which is what we want. But I have to look it up from time to time to confirm that I'm not misremembering.

Reply:
Totally agree

Reply:
That is confusing